syllabus/5-Substrate/1-Intro-to-Substrate_Slides.md

```diff
@@ -8,20 +8,15 @@ duration: 60 minutes
 
 ---
 
-## Before Going Any Further 🛑
-
-While I speak, please clone `substrate`, and run `cargo build && cargo build --release`.
-
-> https://github.com/paritytech/substrate
-
----
-
 ## About These Lectures and Lecturer
 
 - Ground-up, low-level, but hands-on.
 - Intentionally avoiding FRAME, but giving you the tools to be successful at it.
-- Narratives above facts all.
 - Interrupts and questions are always welcome.
+- Narratives above facts all.
```

Comment:
Not sure the grammar is right here.

I think you mean you will focus on explaining the high level narratives and thinking that inform decisions rather than the facts as they are right now. Great strategy.

Reply:
fixed.